### ISSUE TITLE
Index - Link to Concepts/Loader Docs

### ISSUE THREAD

**Issue by joshmosh** [webpack/webpack.js.org#1300]
I recently have made the move to Webpack from the Rails asset pipeline and found myself incredibly confused when getting familiar with the concepts. The first experience I had with Webpack was actually with Rails 5.1 via the Webpacker gem and fell in love. Since I enjoyed the experience so much I thought I would go ahead and try out setting up things on my own and found myself incredibly confused when getting started. I'm not sure if this is up for debate or not, but I'd like to suggest a slightly different approach to getting started with Webpack.

I imagine most people do what I do and do a DuckDuckGo or Google search for Webpack, visit the homepage and click the getting started page. I spent hours trying to figure out how to use a loader because the getting started guide doesn't really explain. It wasn't until I heard from Sean Larkin that the concepts page existed to explain loaders and plugins. From that moment on I've been so happy because if I wouldn't have found the concepts page I would have gone back to using the Webpacker gem and never learning how to do this all on my own.

I'd like to suggest that concepts are highlighted more on the homepage to help people out that are coming from different backgrounds like myself.

Thanks!

**Comment by skipjack** [comment]
@joshmosh sure, would you like to submit a PR to add a link somewhere on the index page? Did you have any thoughts on where? Also, re the guides -- we're in the process of reviewing, editing, and cleaning them up now (see #1258). We'll keep this in mind as we go and, in fact, after the recent re-ordering of guides in #1273 the [third guide](https://webpack.js.org/guides/asset-management/) does touch on loaders.

**Comment by joshmosh** [comment]
@skipjack Yeah! I have a few ideas of how this could be added to the homepage. I'll definitely keep all this movement in mind before starting. I'll familiarize myself with everything.

I'm looking forward to helping! Thanks!

**Comment by skipjack** [comment]
@joshmosh any updates on this? Happy to help push this forward...

**Comment by joshmosh** [comment]
@skipjack I'm sorry for letting this sit for a little. I just moved and have been spending a lot of my time unpacking and organizing. I should be able to make some progress this week.

**Comment by skipjack** [comment]
No worries... sounds good, thanks.

**Comment by joshmosh** [comment]
@skipjack Time keeps getting away from me. I'm currently swamped with preparations for a work trip all next week. If this is something you would like to take ownership of, or hand off I think that's a great idea so I don't keep this sitting. If you're not in a hurry then I am planning on setting aside time after next week (week of July 24th) to get set up and offer some suggestions.

**Comment by skipjack** [comment]
@joshmosh if I haven't opened a PR by next week please feel free to tackle. I'm running short on time these days as well so any help we can get on this repo is much appreciated. If I do have a chance, I'll drop a comment here when I start so we don't overlap (and you should do the same if you start first).

**Comment by stephan-v** [comment]
It still fathoms me how such a large project can have the absolute worst documentation that I have ever seen.

**Comment by skipjack** [comment]
@stephan-v very helpful, thanks. Care to elaborate on that comment or help contribute?

**Comment by stephan-v** [comment]
You need help? I just spend a minute staring at your huge list of sponsors and backers thinking "How in the fuck could your documentation be this bad with so many supporters". Looks like you get plenty of help. 

Even something as basic of digging through your documentation to figure out in which order multiple loaders run, yields 0 results.

Even the first documentation was better, somehow you managed to make it worse.

**Comment by skipjack** [comment]
> You need help? I just spend a minute staring at your huge list of sponsors and backers thinking "How ... could your documentation be this bad with so many supporters". Looks like you get plenty of help.

First of all, please take a second and calm down. I deleted your comment and will delete any others you post with profanity if you don't relax a bit. 

Look, it's unfortunate you feel that the "first documentation was better" and somehow we've all just managed to make it worse. The fact is, we've actually gotten a lot of feedback on this site, most of it positive, with small comments of what's missing or what can be improved. These things are taken into consideration and addressed. Each section is starting to be reviewed in detail by a team (e.g. #1258 and #1386) and other missing documentation or clarifications are addressed as they come up.

> Even something as basic of digging through your documentation to figure out in which order multiple loaders run, yields 0 results.

The `/api` section of our docs is definitely in need of the most help. With all the offers of consistent content editors in #1380, I think we'll get that assistance soon. While we do have [a lot of contributors](https://github.com/webpack/webpack.js.org/graphs/contributors), it is a lot harder to find consistent ones. I personally work on this in my free time and, if you feel there are a lot of things that can still be improved, then I would suggest you set some time aside and try to help improve them. 

If you feel some documentation on loader ordering is missing please create an issue or PR to resolve it. There is some documentation on [Rule.enforce](https://webpack.js.org/configuration/module/#rule-enforce) but I wasn't able to find anything re executed order by default. They are executed right-to-left in the array they are passed in if I remember correctly.

Again please open another issue or PR to resolve this so we can talk there. The issue you're commenting on has nothing to do with this discussion.

**Comment by TheLarkInn** [comment]
@stephan-v if you need someone to talk to, please feel free to reach out to me on Twitter. If there is something we can be doing better I'd like to hear it from you there in a safe and private medium. 

Second you have received your one and only warning. Any further misconduct across any webpack or webpack-contrib organization repos we will act on through banning or removal of participation with the org.

**Comment by joshmosh** [comment]
Woot woot! Thank you @skipjack! Looks good!
